components/bounty/bounty-hero/price-conversor/modal/controller.tsx

```diff
@@ -37,39 +41,40 @@ export default function PriceConversorModal({
 
   const {state} = useAppState();
 
+  const { data: prices, isFetching, isError } = useCoingeckoPrice([
+    { address: token?.address, chainId: token?.chain_id },
```

Comment:
this can't be `?` -- if we're making the useCoingeckoPrice() we _should_ actually have a token.address and toke.chain_id -- if we don't then the request should never be made - lest we make a get that has no values